test/packaging/ansible/roles/logging/tasks/logging.yaml

```diff
@@ -0,0 +1,52 @@
+---
+
+- name: Test with retry if Fluent-Bit process started
+  command: pgrep -f newrelic.*fluent-bit
+  register: output1
+  ignore_errors: true
+  changed_when: false
+  until: output1.rc == 0
+  retries: 3
+  delay: 5
+
+- name: Wait for 5 seconds before checking Fluent-Bit PID again
+  pause:
+    seconds: 5
+
+- name: Get Fluent-Bit PID
+  command: pgrep -f newrelic.*fluent-bit
+  register: output2
+  ignore_errors: true
+  changed_when: false
+
```

Comment:
When fluentbit is installed but not running (i.e. wrongly configured), it's not being detected and the test passes. Maybe we should somehow detect if it's not running at all and fail if expected_version is not empty?

Reply:
maybe there should be a check for PID != nil os similar one for bash 🤔